### ISSUE TITLE
TV State does not work for automation

### ISSUE THREAD

**Issue by GEORGE-VASS** [ollo69/ha-samsungtv-smart#100]
**Describe the bug**
When using an automation for state "TV" it won't work. but with state org.tizen.netflix-app the automation will trigger.

**Expected behavior**
When turning off Netflix and the state goes from  Netflix to tv and action must be run.

What could the simple tv State be?
my tv is AU7172UXX

**Comment by mendoncart** [comment]
Try to monitor the attribute media_title.

**Comment by Ey3scr34m** [comment]
> Try to monitor the attribute media_title.

This used to work for me, but it does not work anymore. Has anything changed?

**Comment by 3raser95** [comment]
I am facing a similar issue. I have an automation that checks whether I'm watching a specific channel and it used to work by checking {{ states.media_player.samsung_q60_series_65_2.attributes["media_title"] }} but it hasn't been working for the last month or so. It used to report the channel name but now it gives an error **UndefinedError: 'homeassistant.util.read_only_dict.ReadOnlyDict object' has no attribute 'media_title'**. Also in the developer tools in the attributes of the device there are no media_title or media_channel attributes that used to be there before.

**Comment by Ey3scr34m** [comment]
Actually I think that I found the problem or at least how to get the missing attributes back. You have to open your smart things app from time to time, so that it kind of syncs your TV.

**Comment by 3raser95** [comment]
Yeah I actually found out that the TV had some sort of error in smart things where it was showing there but it reported as offline always even when the TV was on and connected to the internet. All the controls worked still through HA like changing channel and so forth but I'm guessing the smart things API wasn't working? I now fixed it by removing the TV from smart things and adding it back and now the "media_title" is working again.
